checker/tests/src/tests/NullnessIgnoreRawTypesTest.java

```diff
@@ -0,0 +1,27 @@
+package tests;
+
+import org.checkerframework.framework.test.CheckerFrameworkTest;
+
+import java.io.File;
+
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Tests for the Nullness Checker on raw types using the ignoreRawTypeArguments option
+ */
+public class NullnessIgnoreRawTypesTest extends CheckerFrameworkTest {
+
+    public NullnessIgnoreRawTypesTest(File testFile) {
+        super(testFile,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
```

Comment:
Both of these tests only use the freedom-before-commitment system. Should they also test with rawness?